apps/console/src/features/applications/components/forms/inbound-passive-sts-form.tsx

```diff
@@ -210,146 +197,34 @@ export const InboundPassiveStsForm: FunctionComponent<InboundPassiveStsFormProps
-                        <Divider/>
-                    </Grid.Column>
-                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 10 }>
-                        <Heading as="h5">
-                            {
-                                t("console:develop.features.applications.forms." +
-                                    "advancedConfig.sections.certificate.heading") }
-                        </Heading>
-                        <Field
-                            label={
-                                t("console:develop.features.applications.forms." +
-                                    "advancedConfig.sections.certificate.fields.type.label")
-                            }
-                            name="type"
-                            default={ CertificateTypeInterface.JWKS }
-                            listen={
-                                (values) => {
-                                    setPEMSelected(values.get("type") === "PEM");
-                                }
-                            }
-                            type="radio"
-                            value={ certificate?.type }
-                            children={ [
-                                {
-                                    label: t("console:develop.features.applications.forms." +
-                                        "advancedConfig.sections.certificate.fields.type.children.jwks.label"),
-                                    value: CertificateTypeInterface.JWKS
-                                },
-                                {
-                                    label: t("console:develop.features.applications.forms." +
-                                        "advancedConfig.sections.certificate.fields.type.children.pem.label"),
-                                    value: CertificateTypeInterface.PEM
-                                }
-                            ] }
+                    
+                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                        <ApplicationCertificateWrapper
+                            protocol={ SupportedAuthProtocolTypes.WS_FEDERATION }
+                            deleteAllowed={ !(initialValues?.idToken?.encryption?.enabled) }
+                            reasonInsideTooltipWhyDeleteIsNotAllowed={ (
+                                <Fragment>
+                                    <Trans
+                                        i18nKey={ "console:develop.features.applications.forms" +
+                                        ".inboundOIDC.sections.certificates.disabledPopup" }
+                                    >
+                                        This certificate is used to encrypt the <Code>id_token</Code>. First, you need
+                                        to disable <Code>id_token</Code> encryption to proceed.
+                                    </Trans>
+                                </Fragment>
+                            ) }
```

Comment:
Pass an empty fragment here.

```suggestion
                            reasonInsideTooltipWhyDeleteIsNotAllowed={ <Fragment /> }
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/2977/commits/6d98edaa073d860b55016e33cf784757f1d4a8b2